### PR TITLE
New plugin structure

### PR DESCRIPTION
**Pull request by fonsecareyna82**
Defining the module to convert it as a PIP

### CODE REVIEW

File: localrec/__init__.py [hunk 1]
```diff
@@ -30,34 +30,33 @@
 """
 import os
 import pyworkflow.em
-from bibtex import _bibtex # Load bibtex dict with references
-from opic.constants import *
-from opic.convert import *
+from pyworkflow.utils import Environ
 
+from bibtex import _bibtex # Load bibtex dict with references
```
Comment (delarosatrevin) [inline localrec/__init__.py]: This is no longer needed. As far as it is named "bibtex.py" Scipion will grab the references from there. ;)

File: localrec/convert.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
+# **************************************************************************
+# *
+# * Authors:     Josue Gomez Blanco (josue.gomez-blanco@mcgill.ca)
```
Comment (delarosatrevin) [inline localrec/convert.py]: Maybe add yourself as author?

File: localrec/protocols/protocol_localized.py [hunk 3]
```diff
@@ -0,0 +1,211 @@
+# *****************************************************************************
+# *
+# * Authors:     Josue Gomez Blanco (josue.gomez-blanco@mcgill.ca)
```
Comment (delarosatrevin) [inline localrec/protocols/protocol_localized.py]: Same here.

File: localrec/bibtex.py [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+# * along with this program; if not, write to the Free Software
+# * Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA
+# * 02111-1307  USA
+# *
+# *  All comments concerning this program package may be sent to the
+# *  e-mail address 'scipion@cnb.csic.es'
+# *
+# **************************************************************************
+
+_bibtexStr = """
```
Comment (delarosatrevin) [inline localrec/bibtex.py]: This is the old way to specify bibtex.

**Comment by delarosatrevin** [comment]
@pconesa @yaizar @fonsecareyna82 , I think @vabrishami will maintain this plugin from now on. He is working with @JuhaHuiskonen , the original developer, so I will propose that future changes to the plugin should be discussed with them. (This case is just the plugin structure, but just for the future).

**Comment by pconesa** [comment]
Great! Many thanks @vabrishami for helping with this. Please, count with us for any help you might need.

**Pull request merged**